src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenOnBehalfOfParameterBuilder.cs

```diff
@@ -161,6 +161,20 @@ public AcquireTokenOnBehalfOfParameterBuilder WithCcsRoutingHint(string userName
 
             this.WithExtraHttpHeaders(ccsRoutingHeader);
             return this;
+        }
+
+        /// <summary>
+        /// Only affects <see cref="ILongRunningWebApi.InitiateLongRunningProcessInWebApi(IEnumerable{string}, string, ref string)"/>.
+        /// When enabled, mimics MSAL 4.50.0 and below behavior - checks in cache for cached tokens first, 
+        /// and if not found, then uses user assertion to request new tokens from AAD.
+        /// When disabled (default behavior), doesn't search in cache, but uses the user assertion to retrieve tokens from AAD.
+        /// </summary>
+        /// <param name="searchInCache">Whether to match on cached OBO assertion hash.</param>
+        /// <returns>The builder to chain the .With methods</returns>
+        public AcquireTokenOnBehalfOfParameterBuilder WithSearchInCacheForLongRunningProcess(bool searchInCache = true)
```

Comment:
Not sure this is something we should advertise since it makes the oth api obsolete. I think it is better to hide this with EditorBrowsable = false